### PR TITLE
Added a cron test for disabled all-tokenization

### LINKED ISSUE #3743: Add a cron test when `all-tokenization` is disabled
To avoid having this issue again (https://github.com/meilisearch/meilisearch/pull/3687), we would like to add a test in our CI running every week.

A new job should be created into this file: https://github.com/meilisearch/meilisearch/blob/main/.github/workflows/test-suite.yml
The name of the job should be `test-disabled-tokenization`.
This job should only run for the event `schedule`.
The following command should be done, in two separated steps:
```bash
cargo tree -f '{p} {f}' -e normal --no-default-features | grep lindera -vqz
```
and then
```bash
cargo tree -f '{p} {f}' -e normal | grep lindera -qz
```

You can take inspiration from this already existing cron test job: https://github.com/meilisearch/meilisearch/blob/918ce1dd674f99a41545c50272507a275ca53baf/.github/workflows/test-suite.yml#L80-L106

### PR DESCRIPTION
**Pull request by roy9495**
# Pull Request

## Related issue
Fixes #3743

## What does this PR do?
- This  cron test checks if  all-tokenization is disabled or not

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
Hello!
thank you for the PR

you have to add this missing step

```yml
- uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
          override: true
```

otherwise the command with `cargo` will not work

**Comment by roy9495** [comment]
I updated the cron job . But now there are two commits . Do you want me to squash them?

**Comment by curquiza** [comment]
No that's ok @roy9495, no need to squash. Thank you for your reactivity

File: .github/workflows/test-suite.yml [hunk 1]
```diff
@@ -105,6 +105,29 @@ jobs:
...
+      - uses: actions/checkout@v3
+      - name: Install needed dependencies
+        run: |
+          apt-get update
+          apt-get install --assume-yes build-essential curl
+      - uses: actions-rs/toolchain@v1 
+        with: 
+          toolchain: stable 
+          override: true 
+      - name: Run cargo tree without default features
```
Comment (curquiza) [inline .github/workflows/test-suite.yml:124]:
```suggestion
      - name: Run cargo tree without default features and check lindera is not present
```

File: .github/workflows/test-suite.yml [hunk 2]
```diff
@@ -105,6 +105,29 @@ jobs:
...
+        run: |
+          cargo tree -f '{p} {f}' -e normal --no-default-features | grep lindera -vqz
+      - name: Run cargo tree with default features
```
Comment (curquiza) [inline .github/workflows/test-suite.yml:127]:
```suggestion
      - name: Run cargo tree with default features and check lindera is pressent
```

File: .github/workflows/test-suite.yml [hunk 3]
```diff
@@ -105,6 +105,29 @@ jobs:
           command: test
           args: --workspace --locked --release --all-features
 
+  test-disabled-tokenization:
+    name: Test Disabled Tokenization
```
Comment (curquiza) [inline .github/workflows/test-suite.yml:109]:
```suggestion
    name: Test disabled tokenization
```

**Review by curquiza — APPROVED** [review]
Thanks again @roy9495 for the PR

bors merge

**Comment by curquiza** [comment]
bors merge-

**Comment by curquiza** [comment]
bors merge

**Comment by curquiza** [comment]
bors help

**Comment by curquiza** [comment]
Hello @roy9495 
I'm sorry, I don't know why, there is an issue with our merging tool. It says the branch does not exist, but I can see your branch on your forked repository so I really don't understand.

Can you help us and squash the commits? 🙏 
I will try to merge again, and if it does not work I will cherry-pick this only commit into a new branch and merge it.

Thank you in advance

**Comment by roy9495** [comment]
Certainly, I will be sure to do that.

**Comment by roy9495** [comment]
Sorry for the delay

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
bors try-

File: .github/workflows/test-suite.yml (hunk 3 above)
Comment (curquiza) [inline .github/workflows/test-suite.yml:109]:
```suggestion
    name: Test disabled tokenization
```

File: .github/workflows/test-suite.yml (hunk 1 above)
Comment (curquiza) [reply .github/workflows/test-suite.yml:124]:
```suggestion
      - name: Run cargo tree without default features and check lindera is not present
```

File: .github/workflows/test-suite.yml [hunk 4]
```diff
@@ -105,6 +105,29 @@ jobs:
...
+      - uses: actions/checkout@v3
+      - name: Install needed dependencies
+        run: |
+          apt-get update
+          apt-get install --assume-yes build-essential curl
+      - uses: actions-rs/toolchain@v1 
+        with: 
+          toolchain: stable 
+          override: true 
+      - name: Run cargo tree without default features
```
Comment (curquiza) [inline .github/workflows/test-suite.yml:124]:
```suggestion
      - name: Run cargo tree without default features and check lindera is not present
```

File: .github/workflows/test-suite.yml [hunk 5]
```diff
@@ -105,6 +105,29 @@ jobs:
...
+          apt-get update
+          apt-get install --assume-yes build-essential curl
+      - uses: actions-rs/toolchain@v1 
+        with: 
+          toolchain: stable 
+          override: true 
+      - name: Run cargo tree without default features and check lindera is not present
+        run: |
+          cargo tree -f '{p} {f}' -e normal --no-default-features | grep lindera -vqz
+      - name: Run cargo tree with default features
```
Comment (curquiza) [inline .github/workflows/test-suite.yml:127]:
```suggestion
      - name: Run cargo tree with default features and check lindera is pressent
```

**Review by curquiza — APPROVED** [review]
bors merge

**Comment by curquiza** [comment]
Sorry I don't know why it still does not work.

Replaced by https://github.com/meilisearch/meilisearch/pull/3779

@roy9495, I cherry-picked your commits, so you will still be a contributor of Meilisearch 😄

**Comment by roy9495** [comment]
Thank you

**Comment by curquiza** [comment]
Thank you for your work and time ❤️

**Pull request closed without merging**